vnext/ReactWindowsCore/V8JSIRuntimeHolder.cpp

```diff
@@ -164,6 +164,11 @@ std::shared_ptr<facebook::jsi::Runtime> V8JSIRuntimeHolder::getRuntime() noexcep
 void V8JSIRuntimeHolder::initRuntime() noexcept {
   v8runtime::V8RuntimeArgs args{};
 
+  if (debuggerPort_ > 0) {
+    args.inspectorPort = debuggerPort_;
```

Comment:
>inspectorPort [](start = 9, length = 13)

Why wouldn't we want to assign 0 to inspectorPort? Does it have different default invalid value? eg -1?

Reply:
The default is different (8888). There's also no error checking and would try to open TCP port 0 (will be fixed in future updates to v8jsi).

---
In reply to: [398237781](https://github.com/microsoft/react-native-windows/pull/4424#discussion_r398237781) [](ancestors = 398237781)